src/main/java/cd/go/contrib/elasticagent/executors/GetPluginConfigurationExecutor.java

```diff
@@ -30,25 +27,21 @@
 import static cd.go.contrib.elasticagent.utils.Util.GSON;
 
 public class GetPluginConfigurationExecutor implements RequestExecutor {
-    public static final Field GO_SERVER_URL = new GoServerUrlField("go_server_url", "Go Server URL", false, "0");
-    public static final Field AUTOREGISTER_TIMEOUT = new PositiveNumberField("auto_register_timeout", "Agent auto-register Timeout (in minutes)", "10", true, false, "1");
-    public static final Field MAXIMUM_PENDING_PODS_COUNT = new PositiveNumberField("pending_pods_count", "Maximum Pending Kuberneted Pods Count", "10", true, false, "2");
-    public static final Field KUBERNETES_CLUSTER_URL = new SecureURLField("kubernetes_cluster_url", "Kubernetes Cluster URL", true, "3");
-    public static final Field KUBERNETES_CLUSTER_USERNAME = new Field("kubernetes_cluster_username", "Kubernetes Cluster Username", null, false, false, "4");
-    public static final Field KUBERNETES_CLUSTER_PASSWORD = new Field("kubernetes_cluster_password", "Kubernetes Cluster Password", null, false, true, "5");
-    public static final Field KUBERNETES_CLUSTER_CA_CERT = new Field("kubernetes_cluster_ca_cert", "Kubernetes Cluster CA Certificate", null, false, true, "6");
     public static final Map<String, Field> FIELDS = new LinkedHashMap<>();
+    public static final Field GO_SERVER_URL = new GoServerUrlField("go_server_url", "GoCD server URL", false, "0");
+    public static final Field AUTO_REGISTER_TIMEOUT = new PositiveNumberField("auto_register_timeout", "Agent auto-register timeout (in minutes)", "10", false, false, "1");
+    public static final Field MAX_PENDING_PODS = new PositiveNumberField("pending_pods_count", "Maximum pending pods", "10", false, false, "2");
+    public static final Field OAUTH_TOKEN = new NonBlankField("oauth_token", "Oauth token", true, "3");
+    public static final Field CLUSTER_URL = new SecureURLField("kubernetes_cluster_url", "Cluster URL", true, "4");
```

Comment:
Cluster URL need not be secure right?  I see that the secure URL field hardcodes secure metadata as false. Rename the class?

Reply:
Secure means it is `https`. I will rename that class.